discovery-provider/src/queries/get_disbursed_challenges_amount.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+
+from sqlalchemy import BigInteger, cast, func
+from sqlalchemy.orm import Session
+
+from src.models.rewards.challenge_disbursement import ChallengeDisbursement
+
+
+def get_disbursed_challenges_amount(
+    session: Session, challenge_id: str, since: datetime
+):
+    """
+    Gets the amount disbursed for a given challenge id since the specified timestamp.
+    """
+    amount_disbursed = (
+        session.query(func.sum(cast(ChallengeDisbursement.amount, BigInteger)))
```

Comment:
nit: is the bigint cast necessary if we're capping at 20k and the max track price is 10k? i suppose with abuse there could be many many 10k-cost track purchases.

Reply:
it's a string anyway, so it has to go to Integer, and I figured that it may exceed that someday